stubs/Repository.stub

```diff
@@ -61,4 +61,11 @@ class Repository
      * @phpstan-return class-string<static>
      */
     protected function getRepositoryClassName();
+
+    /**
+     * @phpstan-param array<non-empty-string, mixed> $criteria
+     * @phpstan-param array<non-empty-string, QueryInterface::ORDER_*>|null $orderBy
```

Comment:
Do wo need these both? I think we have put this into a v13 specific stub. What do you think?